README.md

```diff
@@ -125,7 +158,16 @@ To contribute:
   pytest -s
   ```
 
-  We do a test cookiecut of a dummy package and install it to ensure the template setup works.
+- To commit changes
+
+  ```
+  pre-commit run --all-files --color always --verbose
```

Comment:
I think they will need to install the pre-commit hooks:
```
pre-commit install
```
before their first commit (this only needs to be done once). They shouldn't need to do:
```
pre-commit run --all-files --color always --verbose
```
unless they have commit changes to files before installing the hooks

Reply:
Agree with Paul, but the following would be a simpler recommendation
```suggestion
  pre-commit run -a
```